scripts/templates/detector.tf.j2

```diff
@@ -8,9 +8,12 @@
 {%- else -%}
   {%- set type = 'threshold' -%}
 {%- endif -%}
-{%- set filtering_variable = 'base_filtering' -%}
```

Comment:
we must keep the filtering variable.

this is a way to make the signalflow more readable (splitting in different lines, the main line for signal definition is already too long ..)

and this will avoid all the unrelated changes on this PR